test/controllers/calls_controller_test.rb

```diff
@@ -58,10 +58,8 @@ class CallsControllerTest < ActionDispatch::IntegrationTest
 
   describe 'destroy method' do
     it 'should destroy a call' do
-      with_versioning do
-        PaperTrail.request(whodunnit: @user.id) do
-          @patient.calls.create attributes_for(:call)
-        end
+      with_versioning(@user) do
```

Comment:
A bunch of the diffs in here are changing `with_versioning { PaperTrail.request(whodunnit: ...) }` to just `with_versioning(@user)`. I made a minor change to the `with_versioning` helper to decrease lines of code to write, and to delegate thinking about how to handle the user object to the test helper rather than the programmer. (I ran into all sorts of problems and weird behavior around when to close blocks, use @user vs @user.id, etc, and hopefully this cuts down on the headache for us.)

Reply:
nice!